### ISSUE TITLE
Indefinity logon loop for accounts.google.com

### ISSUE THREAD

**Issue by theincogtion** [keepassxreboot/keepassxc-browser#1356]
## Expected Behavior
Google logon (with TOTP)

## Current Behavior
Logon Loop

## Possible Solution
No idea

## Steps to Reproduce (for bugs)
Configure Browser addon to automatically login if only one entry is available and visit accounts.google.com

## Debug info

```
Content Security Policy: "'unsafe-inline'" wird innerhalb von script-src oder style-src ignoriert: nonce-source oder hash-source angegeben 2
sendRemoveListener on closed conduit sponsorBlocker@ajay.app.962072676048 ConduitsChild.jsm:108
Content Security Policy: "'unsafe-inline'" wird innerhalb von script-src oder style-src ignoriert: nonce-source oder hash-source angegeben 2
sendRemoveListener on closed conduit autt@ericgoldman.name.962072676073 ConduitsChild.jsm:108
sendRemoveListener on closed conduit autt@ericgoldman.name.962072676084 ConduitsChild.jsm:108
AbortError: Actor 'Conduits' destroyed before query 'RuntimeMessage' was resolved ConduitsParent.jsm:297
Content Security Policy: "'unsafe-inline'" wird innerhalb von script-src oder style-src ignoriert: nonce-source oder hash-source angegeben 2
sendRemoveListener on closed conduit sponsorBlocker@ajay.app.962072676094 ConduitsChild.jsm:108
Content Security Policy: "'unsafe-inline'" wird innerhalb von script-src oder style-src ignoriert: nonce-source oder hash-source angegeben 2
sendRemoveListener on closed conduit autt@ericgoldman.name.962072676119 ConduitsChild.jsm:108
sendRemoveListener on closed conduit autt@ericgoldman.name.962072676130 ConduitsChild.jsm:108
AbortError: Actor 'Conduits' destroyed before query 'RuntimeMessage' was resolved ConduitsParent.jsm:297
Content Security Policy: "'unsafe-inline'" wird innerhalb von script-src oder style-src ignoriert: nonce-source oder hash-source angegeben 2
sendRemoveListener on closed conduit sponsorBlocker@ajay.app.962072676140 ConduitsChild.jsm:108
Content Security Policy: "'unsafe-inline'" wird innerhalb von script-src oder style-src ignoriert: nonce-source oder hash-source angegeben 2
sendRemoveListener on closed conduit autt@ericgoldman.name.962072676165 ConduitsChild.jsm:108
sendRemoveListener on closed conduit autt@ericgoldman.name.962072676176 ConduitsChild.jsm:108
Content Security Policy: "'unsafe-inline'" wird innerhalb von script-src oder style-src ignoriert: nonce-source oder hash-source angegeben 2
AbortError: Actor 'Conduits' destroyed before query 'RuntimeMessage' was resolved 3 ConduitsParent.jsm:297
sendRemoveListener on closed conduit sponsorBlocker@ajay.app.962072676186 ConduitsChild.jsm:108
Content Security Policy: "'unsafe-inline'" wird innerhalb von script-src oder style-src ignoriert: nonce-source oder hash-source angegeben 2
sendRemoveListener on closed conduit autt@ericgoldman.name.962072676211 ConduitsChild.jsm:108
Webconsole context has changed
sendRemoveListener on closed conduit sponsorBlocker@ajay.app.962072676232 ConduitsChild.jsm:108
sendRemoveListener on closed conduit autt@ericgoldman.name.962072676222 ConduitsChild.jsm:108
Content Security Policy: "'unsafe-inline'" wird innerhalb von script-src oder style-src ignoriert: nonce-source oder hash-source angegeben
```

![google accounts](https://user-images.githubusercontent.com/10852327/122927452-30483f00-d369-11eb-94a8-c11118da017b.gif)

 KeePassXC-Browser-Version: 1.7.8.1
KeePassXC-Version: 2.6.6 
Operating system: Win
Browser: Firefox

**Comment by Staubgeborener** [comment]
+1 Problem still not fixed
```
KeePassXC - 2.7.1
KeePassXC-Browser - 1.7.12
Operating system: Win64
Browser: Mozilla Firefox 100.0
```

**Comment by varjolintu** [comment]
This was fixed in #1637.
